Solutions/Palo Alto - XDR (Cortex)/Detection Queries/High Severity Cortex Incidents. yaml

```diff
@@ -0,0 +1,46 @@
+id: be8325a4-530c-484d-b301-e01be23395e2
+name: Palo Alto XDR High Severity Incidents
+description: |
+  'The query checks for all High Log Severity Alerts and fire incidents in Cortex to return information of the entities.'
+severity: High
+requiredDataConnectors:
+  - connectorId: PaloAltoNetworksCortex
+    dataTypes:
+      - PaloAltoNetworksCortex
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1562
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "Palo Alto Networks"
+  | where DeviceProduct == "Cortex XDR"
+  | where LogSeverity has_any (2,3,4,5,6)
+  | sort by TimeGenerated
+entityMappings:
+  - entityType: Account
+    fieldMappings:
+      - identifier: FullName
+        columnName: SourceUserName
+  - entityType: Host
+    fieldMappings:
+      - identifier: FullName
+        columnName: SourceHostName
+  - entityType: File
+    fieldMappings:
+      - identifier: Name
+        columnName: FileName
+  - entityType: File
+    fieldMappings:
+      - identifier: Name
+        columnName: FilePath
+   - entityType: URL
```

Comment:
There is an extra space in this line which is leading to yaml validation failure.